modules/geo-layers/src/h3-layers/h3-hexagon-layer.js

```diff
@@ -22,9 +23,27 @@ function getHexagonCentroid(getHexagon, object, objectInfo) {
   return [lng, lat];
 }
 
-function h3ToPolygon(hexId) {
+function scalePolygon(hexId, vertices, factor) {
+  const [lat, lng] = h3ToGeo(hexId);
+  // get distinct vertex objects,
+  // `h3ToGeoBoundary` returns same array object for first and last vertex (closed polygon)
+  const uniqueVertices = [...new Set(vertices)];
```

Comment:
- Don't use spread operator in this repo
- This is unnecessary. the construction of new objects is more expensive than transforming one additional point. `h3ToPolygon` should consistently return the same format regardless of `coverage`.

Reply:
`h3ToGeoBoundary` is returning same object (Array object) for first and last vertex , changing one changes the other, I will have WA here (just skip the last), and will open bug for h3-js.

Reply:
I see. I don't think this is necessarily a bug for h3-js, but indeed a catch.